### ISSUE TITLE
Aria-selected is not narrated for the single-select listbox, when selection is NOT moved with focus

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/VFO-standards-support#440]
### Summary

1. Go to https://codesandbox.io/s/m5fjl?module=/example.js
2. Navigate through the selectable list 

### Expected result
JAWS should announce information about aria-selected state

### Actual result
JAWS doesn't announce information about aria-selected 

### Additional Information
**Background about example:**
When we implemented selectable 'listbox' example we consider 
Down Arrow: Moves focus to the next option. Optionally, in a single-select listbox, selection may also move with focus.
Up Arrow: Moves focus to the previous option. Optionally, in a single-select listbox, selection may also move with focus.
from aria 
https://www.w3.org/TR/wai-aria-practices-1.2/#listbox_kbd_interaction

It means that we don't move selection with focus. But user need to press space to do selection. 

### Comment
When I tried the same example with Firefox then JAWS was narrating:
> Irving Kuhic 7:26:56 AM Program the sensor to the SAS alarm through the haptic SQL card! selected
> 1 of 3

Does it mean that JAWS is getting maybe accessibility tree from chromium without the information about aria-selected state? 

#### JAWS version and build number
2020.20006.12 ILM

#### Operating System and version
Win 10 Enterprise
10.0.19041 Build 19041

#### Browser and version:
Chrome
Version 85.0.4183.83 (Official Build) (64-bit)

**Comment by JAWS-test** [comment]
> Does it mean that JAWS is getting maybe accessibility tree from chromium without the information about aria-selected state?

No, Chrome transmits aria-selected correctly. See the Accessibility tab in the developer tools (F12)

**Comment by JAWS-test** [comment]
See: https://github.com/w3c/aria-practices/issues/1513

**Comment by frex65** [comment]
Bug confirmed using JAWS 2020.2008.24 and Chrome 85.0.4183.121. If the JAWS "List item say" is set to "Selected" or "Both", the option with focus is reported as selected, even if a different option is selected.

**Comment by Wildebrew** [comment]
This is still a problem with Chrome and Edge
(latest browser versions with newest version of Jaws 2020))

* Jaws does not announce selection changes in Chrome or Edge
* Jaws has a weird way of announcing selected state in Chrome/Edge

## To reproduce
1. Visit the [APG 1.2 rearrangeable listbox example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/listbox/listbox-rearrangeable.html)
2. Open the "Available Upgrades" listbox in example 2 (it is multi-selectable).

### Chrome

Grade: B-

Action: Press spacebar to select or de-select an option in the listbox
* Expectede: Jaws says the word "selected" or "not selected" based on the new selection state
* Actual:  Jaws says nothing

Navigating through the listbox options with arrow keys
* Expected, Jaws says "selected" for selected options, and optionally "not selected" for unselected ones
* Actual: Jaws says nothing for selected options and "not selected" for unselected ones.
(by "nothing" I mean that Jaws only announces the text of the option, it does not announce its state as "selected").

### Edge:
Grade: B-

Exactly like Chrome with one exception.
If you navigate to a selected option with the arrow keys and press spacebar, Jaws announces "not selected", I cannot trigger any other state change announcement.

### Firefox

Grade: B+

Action: Press spacebar to select or de-select an option
* Expectede: Jaws says the word "selected" or "not selected" based on the new selection state
* Actual:  Jaws says "selected" when you select the option, but nothing when you de-select it

Navigating through the listbox options with arrow keys
* Expected, Jaws says "selected" for selected options, and optionally "not selected" for unselected ones
* Actual: Jaws says the word "checkmark" then the option text, then "selected" for selected options, but says "not selected" + the text of the option for unselected options
(The word "checkmark" probably comes from the CSS).

While a bit odd at first, this is acceptable.
